lib/game.rb

```diff
@@ -0,0 +1,101 @@
+require_relative './Board'
```

Comment:
- [ ] **Hint2:** You wrote your requires with a capital letter but your files have lower case, so it doesn't find these "capital letter files"

Reply:
DONE